src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -633,6 +633,125 @@
       ]
     }
   },
+  {
+    "name": "Kubernetes_Upgrade_Non-Vulnerable_Version",
+    "description": "Kubernetes Services should be upgraded to a non-vulnerable Kubernetes version",
+    "recommendation": "Upgrade to a non-vulnerable Kubernetes version (1.11.9+, 1.12.7+, 1.13.5+, and 1.14.0+)",
+    "helpUri": "https://github.com/Azure/template-analyzer/blob/main/docs/built-in-bpa-rules.md/#kubernetes-services-should-be-upgraded-to-a-non-vulnerable-kubernetes-version",
+    "evaluation": {
+      "resourceType": "Microsoft.ContainerService/managedClusters",
+      "allOf": [
+        {
+          "not": {
```

Comment:
These could also be simplified with something like this for each:

```
{
  "not": {
    "path": "properties.kubernetesVersion",
    "regex": "^1.11.[0-8]*"
  }
}
```
Can't think of a way right now to avoid having three different regex for 11, 12 and 13